React/CoreModules/RCTLogBox.mm

```diff
@@ -72,11 +72,95 @@ - (void)show
 
 @end
 
+#else // [TODO(macOS ISS#2323203)
+
+@interface _RCTLogBoxView : NSView
```

Comment:
This _RCTLogBoxView isn't actually used.   Was this an experiment before putting the isFlipped fix in RCTSurfaceView?

Reply:
Whoops, correct. Thanks!